lib/machete/matchers/unix_time_matcher.ex

```diff
@@ -0,0 +1,119 @@
+defmodule Machete.UnixTimeMatcher do
+  @moduledoc """
+  Defines a matcher that matches integers that represent Unix time in milliseconds
+  """
+
+  import Machete.Mismatch
+
+  defstruct exactly: nil, roughly: nil, before: nil, after: nil
+
+  @typedoc """
+  Describes an instance of this matcher
+  """
+  @opaque t :: %__MODULE__{}
+
+  @typedoc """
+  Describes the arguments that can be passed to this matcher
+  """
+  @type opts :: [
+          {:exactly, Time.t()},
```

Comment:
Shouldn't these types be `integer`?

Reply:
Yes it should! Fixed in a98bf50b8e87db1599afe7ef3b911fdfd69aacf2